go.mod

```diff
@@ -59,6 +60,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/obfuscate v0.35.0-rc.4
 	github.com/DataDog/datadog-agent/pkg/otlp/model v0.35.0-rc.4
 	github.com/DataDog/datadog-agent/pkg/quantile v0.35.0-rc.4
+	github.com/DataDog/datadog-agent/pkg/remoteconfig/client v0.35.0-rc.4
```

Comment:
Note: this isn't valid (`pkg/remoteconfig/client` doesn't have a `v0.35.0-rc.4` version), but it's fine: we don't guarantee modules stability outside of RCs and releases, and this will be replaced by a legal version name once `7.36.0-rc.1` is built.

Reply:
I was unsure about what to put here, nice to hear it's fine 👍🏻